include/MidiController.h

```diff
@@ -48,6 +48,7 @@ class MidiController : public Controller, public MidiEventProcessor
 {
 	Q_OBJECT
 public:
+	static const int NONE = -1;
```

Comment:
Could be `constexpr`:
```suggestion
	static constexpr int NONE = -1;
```

Reply:
done, updated